spec/queries/fees_query_spec.rb

```diff
@@ -0,0 +1,381 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe FeesQuery, type: :query do
+  subject(:fees_query) { described_class.new(organization:, pagination:, filters:) }
+
+  let(:organization) { create(:organization,) }
```

Comment:
Style/TrailingCommaInArguments: Avoid comma after the last parameter of a method call, unless each item is on its own line. (https://rubystyle.guide#no-trailing-params-comma)